SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialMetrics.cpp

```diff
@@ -450,14 +455,53 @@ void USpatialMetrics::SpatialExecServerCmd(const FString& ServerName, const FStr
 			switch (ServerCommand)
 			{
 			case SpatialMetricsPrivate::EServerCommands::StartInsights:
-#if ENGINE_MINOR_VERSION >= 26
-				FTraceAuxiliary::UpdateTraceCapture(*Args);
+			{
+#if ENGINE_MINOR_VERSION < 26
+				UE_LOG(LogSpatialMetrics, Warning,
+					   TEXT("SpatialExecServerCmd: Failed to execute server StartInsights command. Command only available prior to 4.26."));
+#elif !UE_TRACE_ENABLED
+				UE_LOG(LogSpatialMetrics, Warning,
+					   TEXT("SpatialExecServerCmd: Failed to execute server StartInsights command. UE_TRACE_ENABLE not defined."));
 #else
+				if (StartInsightsCapture(Args))
+				{
+					FString TraceTimeString;
+					if (FParse::Value(*Args, TEXT("-tracetime="), TraceTimeString))
+					{
+						int32 TraceTime = FCString::Atoi(*TraceTimeString);
```

Comment:
If you have an empty ' ' in here you will get 0 back, is that what you want?

Reply:
Oh interesting. I'm not sure. What would you expect if you put `-tracetime=` ?

Reply:
either a crash or ensure with a sensible default.

Reply:
Basically some way of tracing back this error to the place it actually happened.

Reply:
I think its reasonable to assume that an empty string means a dev or tool person forgot to fill in the string.

Reply:
If this is the first in a string of arguments you might just eat a spacing space as well or consume end padding which sometimes (accidentally) exists when constructing commandline arguments.

Reply:
(I have recently seen this happen in someone else's code, so its not theoretical)

Reply:
Okay I'll check if the value is <= 0, and warn and ignore if so